docs/DeveloperGuide.md

```diff
@@ -169,7 +170,21 @@ the same full name with different contact details but it will be unreasonably ra
 number, email and addresses simultaneously.
 
 ### 4. Flexible Command Keywords.
-Command keywords are currently case-sensitive i.e. `add` is a valid command but `Add` is not. For user convenience, we will make command keywords case-insensitive in a future update by parsing the prefixes differently.
+Command keywords are currently case-sensitive i.e. `add` is a valid command but `Add` is not. For user convenience, we
+will make command keywords case-insensitive in a future update by parsing the prefixes differently.
+
+### 5. No Wrapping for very long ui details
+Currently, if the name, tags or other details are very long, they are cut off in the UI. This will be fixed in a future
+version by enabling wrapping for texts in the UI such that they are more easily viewable.
+
+### 6. Some common names are not allowed to be entered
+Currently, our app does not allow some common Singaporean names with some special characters to be entered. An example
+is "s/o" or "d/o". This will be fixed in a future version because we will change the way we parse the add and edit
```

Comment:
Maybe we can add that:

In the future, we will add support for more name formats which include, but are not limited to:

1. Names with a `/` eg. `s/o` `d/o`
2. Names with an apostrophe eg. `D'Silva`
3. Names with a hyphen eg. `John Lim-Tan`
4. Names with an @ eg. `Tan Xiao Ming @Chen Xiaoming`

To increase the robustness of this planned enhancement 🙂

Reply:
if it becomes this specific, it can be argued that it is indeed 4 separate enhancements